client/load.go

```diff
@@ -5,12 +5,15 @@ import (
 	"os"
 	"strings"
 
+	runconfigopts "github.com/docker/docker/runconfig/opts"
 	gflag "github.com/jessevdk/go-flags"
 )
 
 func (cli *HyperClient) HyperCmdLoad(args ...string) error {
 	var opts struct {
-		Input string `short:"i" long:"input" value-name:"\"\"" description:"Read from a tar archive file, instead of STDIN"`
+		Input string   `short:"i" long:"input" value-name:"\"\"" description:"Read from a tar archive file, instead of STDIN"`
+		Name  string   `short:"n" long:"name" value-name:"\"\"" description:"Name to use when loading OCI image layout tar archive"`
```

Comment:
this option is used as the name of loaded image. and this option is consist with https://github.com/docker/docker/pull/26369